### ISSUE TITLE
How to count kills for quest?

### ISSUE THREAD

**Issue by kidbreaker22** [OpenKore/openkore#1103]
Please refer to the following resources for help using OpenKore:
 faqs - http://openkore.com/index.php/Frequently_Asked_Questions (such as "Packet Tokenizer")
 forum - http://forums.openkore.com
 documentation wiki - http://openkore.com/index.php/Main_Page

* Before submitting an issue, ensure you are on the latest release
* Check if an issue already exists for your request
* Please post in English if possible

###### Summary: 
Trying to make a macro for kills during quest but I keep failing. I added killcountFix in sys.txt but I don't know if that would help.

###### Affected configuration(s)/ file(s):
```Include relevant code snippets and source code references```
###### Expected Behavior:
Successfully count the current kills for quest
e.g. In my code, $temp1 should equal to 1.

###### Actual Behavior:
Code doesn't work for quest count fix

###### Steps to Reproduce:

![untitled](https://user-images.githubusercontent.com/31119522/30232035-c570ae96-94bb-11e7-9fdc-2c3d0a3468bc.png)

automacro Test {
	eval $::questList->{7122} > 0
	exclusive 1
	timeout 10
	call {
		$temp1 = @eval($::questList->{7122}->missions->{1002}->{count})
		log Poring kills = $temp1
		if ($temp1 == 1) {
			log Poring kills = $temp1
			do ai manual
			log It works!!!!
		}
	}
}

**Comment by alisonrag** [comment]
try:
`$questList->{'questID'}{'missions'}{'mobID'}{'goal'}`

**Comment by kidbreaker22** [comment]
@alisonrag I tried using  `$temp1 = @eval($questList->{7122}{'missions'}{1002}{'count'})`
like you said and it still gave me an error.

**Comment by lututui** [comment]
That no longer works. EventMacro has QuestHuntCompleted and QuestHuntOngoing, and other conditions and special keywords related  to quests. You might want to look into it

openkore.com/index.php/EventMacro

**Comment by kidbreaker22** [comment]
@lututui Hi, 
I tried using eventMacro and I got QuestHuntOngoing running. My question is how can I make my macro count the kills so it knows it is done the quest and I can give in the quest? If it is the QuestHuntCompleted then I think I'm doing something wrong in my code.

automacro TESTINGTHISSHIT3 {
	QuestHuntOngoing 7122 1002
	MobNear /Poring/i
	exclusive 1
	call {
		do a $.MobNearLastBinId
	}
}

automacro TESTINGTHISSHIT2 {
	QuestHuntCompleted 7122 1002
	exclusive 1
	call {
		pause 1
		do ai manual
		log It works!!!!
	}
}
